lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -180,6 +180,14 @@ protected function _getTransactionIsolationLevelSQL($level)
     /**
      * {@inheritDoc}
      */
+    public function getListDatabasesSQL()
+    {
+        return 'PRAGMA databases';
```

Comment:
In sqlite the PRAGMA is "database_list" not "databases".
You will need to fix the tests and verify that the ouput of that PRAGMA is comparable to the output of the others platforms.